src/features/duckplayer/util.js

```diff
@@ -244,17 +269,39 @@ export class DomState {
         if (this.loaded) return loadedCallback()
         this.loadedCallbacks.push(loadedCallback)
     }
+}
 
-    /**
-     * @param {Element} element
-     * @param {MutationCallback} callback
-     */
-    onChanged (callback, element = document.body) {
-        const observer = new MutationObserver(callback)
-        observer.observe(element, {
-            subtree: true,
-            childList: true,
-            attributeFilter: ['src']
-        })
+/**
+ * A single place to validate the overflow settings
+ * and set defaults if needed
+ * @param {any} input
+ * @returns {import("../duck-player").OverlaysFeatureSettings}
+ */
+export function validateSettings (input) {
+    return {
+        selectors: {
```

Comment:
Can this not be done by schema validating the config again?

Reply:
yeah it can, but for testing on platforms I wanted to leave the defaults in. I'll remove this before we merge, just need to get some macOS testing done.